AV/VehicleAutomationLevels.rdf

```diff
@@ -221,6 +239,18 @@
 		<AnnotationVocabulary:explanatoryNote xml:lang="en">The DDT fallback-ready user need not supervise a level 3 ADS while it is engaged but is expected to be prepared to either resume the DDT when the ADS issues a request to intervene or to perform the fallback and achieve a minimal risk condition if the failure condition precludes normal operation.</AnnotationVocabulary:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/ConfinedGeographicLocation">
+		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/OperatingCondition"/>
+		<rdfs:label xml:lang="en">confined geographic location</rdfs:label>
+		<skos:definition xml:lang="en">A region (point, line, or area) on Earth&apos;s surface or elsewhere.</skos:definition>
+	</owl:Class>
```

Comment:
Why are some specialisations of OperatingCondition its instances, e.g., PoorRoadMaintenance, while others are its subtypes, e.g., CorporateCampus?

Reply:
My point is that some ODD attributes (such as road type, weather type, traffic type and time of day) have only a few well-defined values. And in such cases, I define instances, not classes. As a result, it is possible to describe a particular :ODD123 as follows:

:ODD123 :hasOperatingCondition :Freeway, :FairWeather, :StopAndGoTraffic, :Daylight .

where all :hasOperatingCondition property values are instances (and not classes).

Reply:
> My point is that some ODD attributes (such as road type, weather type, traffic type and time of day) have only a few well-defined values. And in such cases, I define instances, not classes. As a result, it is possible to describe a particular :ODD123 as follows:
> 
> :ODD123 :hasOperatingCondition :Freeway, :FairWeather, :StopAndGoTraffic, :Daylight .
> 
> where all :hasOperatingCondition property values are instances (and not classes).

The number of instances is not relevant for the ontological interpretation of a given concept.
Unless some evidence suggests otherwise, I recommend modelling all specialisations of OperatingCondition as its subclasses.